requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.11.22 # pyup: >=1.11,<1.12
+Django==2.0.4
```

Comment:
Why not 2.2? Do all the dependencies still work with this django version?

Reply:
Will upgrade to 3.x anyways, so I just chose this version.
Still have to check the dependencies

Reply:
Dependencies still work